src/services/mock/db.json

```diff
@@ -0,0 +1,12 @@
+{
+  "userprofile": {
```

Comment:
This creates `/userprofile`. In theory we can create any endpoint like `/users`, `/graphql`, etc.